### ISSUE TITLE
Can I view .d file extensions in project file view?

### ISSUE THREAD

**Issue by hikaen2** [intellij-dlanguage/intellij-dlanguage#494]
I'm using IntelliJ IDEA 2019.1 (Community Edition) with intellij-dlanguage.
Can I view .d file extensions in project file view?
![Screenshot_20190504_230837](https://user-images.githubusercontent.com/4222502/57180370-f4209e80-6ec2-11e9-8ec5-508e3f75c7ab.png)

**Comment by pirocks** [comment]
I could have sworn there was already an issue for this, but I can't find it so leaving as is. This is somewhat consistent with Java/Kotlin/other languages supported by Intellij, however I agree that an option to change this would be ideal. I am not currently aware of any documented way of changing this with the intellij openapi, with out changing the actual names of the indexed files, which is not ideal.

**Comment by SingingBush** [comment]
it should be consistent with other languages. We do use different icons for files so there should be a visual cue to distinguish between a `.d` and a `.di` file. I think `package.d` is also treated differently

**Comment by hikaen2** [comment]
> This is somewhat consistent with Java/Kotlin/other languages supported by Intellij

Yes. In Java case, the .java extension is not displayed normally.
but if .java file contains multiple classes, the .java extension will be displayed:
![Screenshot_20190507_202121](https://user-images.githubusercontent.com/4222502/57295802-4271af80-7106-11e9-9fe5-f8d4aaa7fbe6.png)
So I Think it is a class name (not a filename without extension) .

Or in PyCharm, the .py extension is always displayed.
then I want to see .d extension.
